### PR TITLE
AC: extend image quality metrics support

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/metrics/README.md [hunk 1]
```diff
@@ -117,7 +117,19 @@ More detailed information about calculation segmentation metrics you can find [h
...
+  * `block_size` - block size for blocked effect factor. Optional, default `8`.
 * `ssim` - [Structural similarity](https://en.wikipedia.org/wiki/Structural_similarity). Supported representations: `ImageProcessingAnnotation`, `ImageProcessingPrediction`, `ImageInpaintingAnnotation`, `ImageInpaintingPrediction`, `SuperResolutionAnnotation`, `SuperResolutionPrediction`.
+* `vif` - [Visual Information Fidelity](https://en.wikipedia.org/wiki/Visual_Information_Fidelity).
+  * `sigma_nsq` - variance of the visual noise (default = 2).
+* `lpips` - [Learned Perceptual Image Patch Similarity](https://richzhang.github.io/PerceptualSimilarity/).  Supported representations: `SuperResolutionAnnotation`, `SuperResolutionPrediction`, `ImageProcessingAnnotation`, `ImageProcessingPrediction`, `ImageInpaintingAnnotation`, `ImageInpaintingPrediction`.
+** Metric calculation requires `lpips` package installation.**
+* `color_order` - the field specified which color order `BGR` or `RGB` will be used during metric calculation (Optional. Default value is RGB), used only if you have 3-channel images.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/README.md:129]:
```suggestion
  * `color_order` - the field specified which color order `BGR` or `RGB` will be used during metric calculation (Optional. Default value is RGB), used only if you have 3-channel images.
```

**Comment by eaidova** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/metrics/image_quality_assessment.py [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+            prediction_image = prediction_image[:, :, ::-1]
+        gt_tensor = lpips.im2tensor(annotation_image, factor=self.color_scale / 2)
+        pred_tensor = lpips.im2tensor(prediction_image, factor=self.color_scale / 2)
+        return self.loss(gt_tensor, pred_tensor).item()
+
+    def evaluate(self, annotations, predictions):
+        mean, std = super().evaluate(annotations, predictions)
+        if self.dist_threshold:
+            invalid_ratio = np.sum(np.array(self.magnitude) > self.dist_threshold) / len(self.magnitude)
+            self.meta['names'].append('{}@ratio_greater_{}'.format(self.__provider__, self.dist_threshold))
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/image_quality_assessment.py:418]:
```suggestion
            self.meta['names'].append('ratio_greater_{}'.format(self.dist_threshold))
```

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**